mlflow/ml-package-versions.yml

```
@@ -265,7 +265,7 @@ spacy:
 
   models:
     minimum: "3.4.4"
-    maximum: "3.8.5"
+    maximum: "3.8.6"
```

```suggestion
    maximum: "3.8.4"
```

This is a micro version update. Let's avoid supporting something new.